### DISCUSSION TITLE
Speed Triggers

### DISCUSSION THREAD

**Discussion by YertyArg** [gopro/labs#833] [Feature Requests]
Every time I turn on my camera I have to read the QR to apply the changes. This is so?
I need the changes to be REALLY Permanent. Is this possible?

**Comment by dnewman-gpsw** [comment]
Yes. Just multiple ways to do most things.

**Reply by YertyArg** [reply]
But isn't there a way for this not to happen like this?
That the change is permanent, and that every time I send my camera this option is active?
It would be more practical!!!!

**Reply by dnewman-gpsw** [reply]
Learn about boot commands.  It is discussed on the forum and examples for many scripts.

**Reply by YertyArg** [reply]
Cual es el foro? Me puedas dar el link?

**Reply by dnewman-gpsw** [reply]
This is the forum, where you posted, lots of users running boot commands.  https://github.com/gopro/labs/discussions
